.github/workflows/CI.yml

```diff
@@ -0,0 +1,82 @@
+name: CI
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  Test:
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-20.04
+          - windows-2019
+          - macos-10.15
+        d:
+          - "ldc-1.27.1"
+          - "dmd-2.097.2"
+        meson:
+          - 0.59.1
+        ninja:
+          - 1.10.2
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
is fetch-depth needed? This would always fetch all the commits, not just the currently relevant one, which would slow down CI which might not be needed.

Reply:
```suggestion
```